ci-operator/config/openshift/cluster-api-provider-metal3/openshift-cluster-api-provider-metal3-main.yaml

```diff
@@ -6,6 +6,14 @@ base_images:
 binary_build_commands: make build
```

Comment:
you need to add some the image tag, something like https://github.com/openshift/release/blob/b77be85cd61372cd3dc0e673a70ff04807962a7f/ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml#L6C1-L9C36